gnmi_server/server_test.go

```diff
@@ -107,6 +107,25 @@ func createServer(t *testing.T, port int64) *Server {
 	return s
 }
 
+func createReadServer(t *testing.T, port int64) *Server {
+	certificate, err := testcert.NewCert()
+	if err != nil {
+		t.Errorf("could not load server key pair: %s", err)
```

Comment:
This should be `t.Fatalf(...)` as `t.Errorf(...)` does not stop execution of the test and it will continue despite lack of the certificate.